### DISCUSSION TITLE
SpeedyDex mod vs Settings in world options

### DISCUSSION THREAD

**Discussion by PaulBlay** [AtomicFox556/Cataclysm-EOD#153] [Q&A] [answered]
In the world options there are settings for "Minimum dexterity to increase speed" and "Speed per dexterity point". Are these redundant with the SpeedyDex mod? Should you just pick one of the mod and the settings and does it matter which?

**Comment by AtomicFox556** [comment, marked as answer]
Yeah, that makes SpeedyDex redundant; those two options let you configre the effect yourself.
